drivers/gles3/storage/particles_storage.cpp

```diff
@@ -1136,6 +1149,21 @@ void ParticlesStorage::update_particles() {
 			}
 		}
 
+		if (particles->request_process_time > 0.0) {
+			double frame_time;
+			if (fixed_fps > 0) {
+				frame_time = 1.0 / fixed_fps;
+			} else {
+				frame_time = 1.0 / 30.0;
```

Comment:
I'd document this 30 FPS fallback for particles with fixed FPS set to `0` in the class reference in the `fixed_fps` description, as well as `RenderingServer.particles_set_request_process_time()`.